ts/scripts/publish-bins.sh

```diff
@@ -20,5 +20,5 @@ set -xeuo pipefail
 
 VERSION=${1#*-v};
 
-cd go/.out
+cd ts/typegen/.out
 gsutil cp -- *.tar.gz gs://kpt-functions/v"${VERSION}"
```

Comment:
Oh, this gcs is really new to me. I'm wondering if we still have control of the gcs (otherwise we probably should upload to some CaD owned cloud bucket)